drivers/driver.go

```diff
@@ -42,6 +42,13 @@ type CreateOpts struct {
 	StorageOpt map[string]string
 }
 
+// MountOpts contains optional arguments for Get() methods.
+type MountOpts struct {
+	MountLabel string
```

Comment:
All of the members could do with some documentation as well (e.g. it’s not locally obvious that `MountLabel` is a SELinux label.

(OTOH the documentation of the `Get`/`Mount` methods that now use `MountOpts` could be trimmed and rely on the member documentation here.)